Gruntfile.js

```diff
@@ -18,20 +18,25 @@ module.exports = function (grunt) {
 
     concatConfig.target = {
         src: [
-            'src/scripts/directives.js'
+            'src/scripts/**/directives.js',
+            'src/scripts/**/templates.js'
         ],
         dest: 'dist/scripts.js'
 
     };
 
-    ngtemplatesConfig.target = {
-        cwd: 'src',
-        src: 'templates/type/*.html',
-        dest: 'dist/templates.js',
-        options: {
-            module: 'angular-input-fields'
-        }
-    };
+    var modules = fs.readdirSync('src/templates/aif');
```

Comment:
If you use [`grunt.file.recurse`](http://gruntjs.com/api/grunt.file#grunt.file.recurse) it will ignore dotfiles for you, I think